main/parse.c

```diff
@@ -1158,6 +1189,28 @@ static void lazyInitialize (langType language)
 /*
 *   Option parsing
 */
+#define COUNT(D) (sizeof(D)/sizeof(D[0]))
+
+static void lang_def_flag_file_kind_long (const char* const optflag, const char* const param, void* data)
```

Comment:
Updated rules are:
* function name, type names and struct memeber name in *.h should not use underscore.
* parameter names in function decl in *.h can use underscore.
* in *.c file underscore can be used.